mobile/src/libs/components/player/components/controls/controls.tsx

```diff
@@ -25,6 +30,12 @@ const Controls: React.FC<Properties> = ({ isPlaying: initialIsPlaying }) => {
     handlePlayPause();
   };
 
+  useFocusEffect(
+    useCallback(() => {
+      setIsPlaying(false);
+    }, []),
```

Comment:
Shouldn't `setIsPlaying` be in dependencies? The same with dependencies of `useCallback`'s in other places